RUFAS/routines/manure/manure_treatments/manure_types.py

```diff
@@ -7,11 +7,9 @@ class ManureType(Enum):
 
     The Enum class contains the following members:
     LIQUID: Represents liquid manure.
```

Comment:
```suggestion
    Attributes
    ----------
    LIQUID : str
        Represents liquid manure.
    ...
```
This was how I suggested Pooya format the docstrings for Enums in the Feed Storage module - however, if the proper way to refer to elements of an Enum is "members", maybe it would be more appropriate to put:
```suggestion
    Members
    -------
    LIQUID : str
        Represents liquid manure.
    ...
```

Reply:
I think your first way makes the most sense if we want to go one of these two routes. However, it looks like Numpy doesn't specify anything about how to write Enum class docstrings and in fact in their own repo don't have any Attributes or Members listed:

https://github.com/numpy/numpy/blob/84da5965e28e545cb84f8d3ab8bb2321db4ccd2e/numpy/f2py/symbolic.py#L36

class Language(Enum):
    """
    Used as Expr.tostring language argument.
    """
    Python = 0
    Fortran = 1
    C = 2

Reply:
Nice find, I did not look close enough at the NumPy repo. I think going the first route is best too (using "Attributes") - definitely better than not documenting the elements of an Enum at all.